Sources/ArgumentParser/Usage/HelpGenerator.swift

```diff
@@ -18,19 +18,72 @@ internal struct HelpGenerator {
     struct Element: Hashable {
       var label: String
       var abstract: String = ""
-      var discussion: String = ""
-      
+      var discussion: Discussion = .staticString("")
```

Comment:
thought: Maybe this could be an optional discussion type?

Reply:
Fixed